solutions/silver/cses-1085.mdx

```diff
@@ -71,9 +68,14 @@ int main() {
 	int n, k;
 	cin >> n >> k;
 	vector<ll> arr(n);
-	for (ll &i : arr) { cin >> i; }
+	ll sum = 0, mx = 0;
```

Comment:
i'd rather you use `std::max_element` and `std::accumulate` instead

Reply:
Updated now @SansPapyrus683